Java/syntax_test_java.java

```diff
@@ -687,7 +687,7 @@ public void someMethod(WithParam foo) throws Exception {
 //                                  ^ constant.numeric.java
 
     foo.forEach((k, v) -> {
-//                     ^ storage.type.lambda.java
+//                     ^ storage.type.function.lambda.java
```

Comment:
I agree @FichteFoll. This construct (lambda notation) should be added to the style guide.